modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -1528,20 +1528,24 @@ def fold(self, axis, func):
         -------
         PandasDataframe
             A new dataframe.
-
-        Notes
-        -----
-        The data shape is not changed (length and width of the table).
         """
         new_partitions = self._partition_mgr_cls.map_axis_partitions(
             axis, self._partitions, func, keep_partitioning=True
         )
+        # Folding an operation for pandas.Daframe.Rolling can cause columns
```

Comment:
Should this change really be happening here? I thought `fold` was synonymous to `reduce`, but here it seems like its the same as a `map` full axis?